chpl/chpl-service/src/main/java/gov/healthit/chpl/listing/measure/MeasureCriterionMapDAO.java

```diff
@@ -0,0 +1,18 @@
+package gov.healthit.chpl.listing.measure;
+
+import org.springframework.stereotype.Repository;
+
+import gov.healthit.chpl.dao.impl.BaseDAOImpl;
+
+@Repository
+public class MeasureCriterionMapDAO extends BaseDAOImpl {
+    public MeasureCriterionMapEntity create(Long certificationCriterionId, Long measureId, Long lastUpdateUserId) {
```

Comment:
For stuff like this where we are using a one-time job to create/update data but we don't expect creating/updating to be a regular thing I've put the needed relevant DAO methods in a private inner class of the job so they aren't hanging around in the code after the job is deleted. If you don't think these measure create/update methods (in a few DAO classes in this PR but I'm only commenting on this one) are referenced anywhere else in the code it might be nice to group them in with the job to help us clean up things later.